File: C & C++ PRACTICE CODES/practical sheet 3/code7.c
#include <stdio.h>
void main (){
	int selling_price,cost_price,profit;
	printf("Enter the selling price : ");
	scanf("%d",&selling_price);
	printf("Enter the cost price : ");
	scanf("%d",&cost_price);
	profit=selling_price-cost_price;
	if(profit >0){
	printf("profit is : %d",profit);
	}else if (profit<0){
	printf("loss is : %d",profit*(-1));
	}else{
	printf("neither profit nor loss is : %d",profit);
	}



}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/4/code7.java
import java.util.Scanner;

class IfElse{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter your purseing price : ");
		float x = s.nextFloat();
		System.out.println("enter your selling price : ");
		float y = s.nextFloat();

		float result = y - x;


		if(result >= 0 ){

			System.out.println("profit of "+result);
		}else {

			result *= -1;
			System.out.println("loss of "+result);

		}

	}


}
